src/remote.sh

```diff
@@ -121,15 +121,15 @@ function prepare_host_deploy_dir()
   # We should expect the setup.sh script create the directory $HOME/kw.
   # However, does not hurt check for it and create in any case
   if [[ ! -d "$kw_dir" ]]; then
-    mkdir "$kw_dir"
+    mkdir -p "$kw_dir"
```

Comment:
`.cache` is indeed an ephemeral directory (users might manually delete its contents at any time). To avoid problems in the future, when other commands might want to use cache, what if we always create the dir at kw startup? Not only `.cache/kw` but also `.cache` itself (i.e. `mkdir -p `$HOME/.cache/kw`). Then, we could remove the dir creation from setup.sh (which IMHO, makes sense, since the cache is supposed to exist only when running kw)

It would also be nice to have something like `kw clear-cache`, to delete all `.cache/kw` when the user desires to (for storage saving or debugging, for example)

Reply:
Really nice idea! I add this in the next version